### PR TITLE
Update CIO search article

### PR DESCRIPTION
**Pull request by ascott1**
This includes a few small changes and updates the date of the CIO search article to reflect the upcoming posting.

I also had a stray commit that unpublished the internship listing. This seems like something that should be unpublished, but I'm happy to remove this if others would prefer.

### CODE REVIEW

File: _posts/2016-10-06-next-cio.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ permalink: /articles/searching-for-our-next-cio/
...
+The [Consumer Financial Protection Bureau (CFPB)](http://consumerfinance.gov) is a government agency that helps consumer finance markets work for Americans by making rules more effective, by consistently and fairly enforcing those rules, and by empowering consumers to take more control over their economic lives.  From the Bureau’s founding in the wake of the 2008 financial crisis, it has always been clear that we need to leverage modern technology and take a data-driven approach to achieve these goals.  The CFPB technology team uses and produces [open source code](https://github.com/cfpb), has embraced systems in the public cloud, and follows an [Agile](http://www.agilemanifesto.org/) software development model. Our team is made up of talented individuals from both the public and private sectors, and through CFPB’s [Technology and Innovation Fellows](http://www.consumerfinance.gov/jobs/technology-innovation-fellows/) program, we bring together designers, developers, data scientists, and security professionals to collaborate and solve business problems using technology.
```
Comment (Scotchester) [inline _posts/2016-10-06-next-cio.md]: Unrelated to this change, but http://www.consumerfinance.gov/about-us/careers/technology-innovation-fellows/ is looking kinda funky these days...

File: _posts/2016-10-06-next-cio.md [hunk 2]
```diff
@@ -42,13 +42,12 @@ We are looking for a broad and diverse pool of candidates to apply for the posit
...
     - Proven ability to manage significant change in a complex enterprise environment.
+    - Experience aligning operational capacity and resources (budget, people) to business strategy.
     - Experience inspiring, leading, and managing a large cross-functional team (e.g., 100+ employees/contractors).
 - **Technical experience**
     - Undergraduate or graduate degrees in computer science or similar technical discipline.
     - Experience leading the engineering and delivery of innovative technical solutions across a range of technical domains.
-- **Financial services subject matter expertise**
-    - Experience working in financial services, particularly within consumer financial services (including regulators).
-    - Experience leading compliance, operations, and technology functions within financial services organizations (including regulators).
+    - Experience developing and implementing an enterprise – wide technology strategy .
```
Comment (Scotchester) [inline _posts/2016-10-06-next-cio.md]: Please replace "enterprise – wide" with "enterprise-wide" and remove the space prior to the period.

File: _posts/2016-10-06-next-cio.md (hunk 1 above)
Comment (Scotchester) [inline _posts/2016-10-06-next-cio.md]: Er, http://www.consumerfinance.gov/jobs/technology-innovation-fellows/ I mean.

File: _posts/2016-10-06-next-cio.md (hunk 2 above)
Comment (ascott1) [inline _posts/2016-10-06-next-cio.md]: 👍  bad C&P.

File: _posts/2016-10-06-next-cio.md (hunk 1 above)
Comment (ascott1) [inline _posts/2016-10-06-next-cio.md]: Yeah, yuckkk. I'm going to remove the link.

**Comment by ascott1** [comment]
All requested changes have been addressed @Scotchester. Let's hold off on merging this until a final 👍 from COO.

**Pull request merged**